app/css/openmrs-addonmanager.css

```diff
@@ -488,3 +492,34 @@ input[type=text] {
     background-color: #FFF;
     border: 1px solid #dddddd;
 }
+
+.waiting-modal {
+    display:    none;
+    position:   fixed;
+    z-index:    1000;
+    top:        0;
+    left:       0;
+    height:     100%;
+    width:      100%;
+    background: rgba( 255, 255, 255, .8 ) 
+                url('http://i.stack.imgur.com/FhHRx.gif') 
```

Comment:
Can we make this gif static as opposed to fetching it via an API every time. What do you think?

Reply:
OpenMRS app caches static files, so the image will be fetched only once and stored in the cache, subsequent requests will then be served from the cached file @andela-jomadoye, but I'll make it static so that users who install the app and is running it for the fist time will experience slight increase in load speed.